LuaMenu/widgets/snd_music_lite.lua

```diff
@@ -19,13 +19,8 @@ end
 local playingTrack	-- boolean
 local previousTrack
 local loopTrack	-- string trackPath
-
-local randomTrackList = {
-	"sounds/lobbyMusic/A Magnificent Journey (Alternative Version).ogg",
-	"sounds/lobbyMusic/Dream Infinity.ogg",
-	"sounds/lobbyMusic/Interstellar.ogg",
-	"sounds/lobbyMusic/Tomorrow Landscape.ogg",
-}
+local randomTrackList
+local OPEN_TRACK_NAME
```

Comment:
I am going to change the name of OPEN_TRACK_NAME because it is not what I would call a constant. You can do this before it is pulled if you like.

Reply:
Indeed, I'll fix